armory/baseline_models/pytorch/deep_speech.py

```diff
@@ -0,0 +1,14 @@
+"""
+Automatic speech recognition model
+
+Model contributed by: MITRE Corporation
+"""
+
+from art.estimators.speech_recognition import PyTorchDeepSpeech
+import torch
+
+DEVICE = torch.device("cuda" if torch.cuda.is_available() else "cpu")
+
+
```

Comment:
I removed the preprocessing_fn, as it doesn't fit with the canonical data formats and deep speech already does data normalization in the log-spectrogram domain here:
https://github.com/Trusted-AI/adversarial-robustness-toolbox/blob/dev_1.4.2/art/estimators/speech_recognition/pytorch_deep_speech.py#L585-L589